.gitignore

```diff
@@ -8,6 +8,7 @@ composer.lock
 /vendor
 composer.phar
 .php_cs.cache
+.DS_Store
```

Comment:
This have nothing to do on project `.gitignore`.

This file is OS specific. See: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore